test/import-http.bats

```diff
@@ -73,4 +73,34 @@ function teardown() {
     [ "$(sha reference/nm_orig)" == "$(sha dest/img/rootfs/root/nm)" ]
 }
 
+@test "importing to a dest" {
+    cat > img/stacker1.yaml <<EOF
+centos_base:
+    from:
+        type: oci
+        url: $CENTOS_OCI
+    run: |
+        ls
+EOF
+    cat > img/stacker2.yaml <<EOF
+img:
+    from:
+        type: oci
+        url: $(pwd)/oci:centos_base
+    import:
+        - path: http://network-test.debian.org/nm
```

Comment:
Could you just re-use `https://www.cisco.com/favicon.ico` that is used in several other places in tests ? that way we dont' needlessly extend the infrastructure that tests depend on.

Reply:
Ok, done